### ISSUE TITLE
Flux Helm releases: install retries exhausted

### ISSUE THREAD

**Issue by gberche-orange** [orange-cloudfoundry/paas-templates#2163]
### Expected behavior

* As a paas-templates operator
* In order to recover from Flux Helm releases: install retries exhausted
* I need tooling to automate forcing retries, possibly in the right order

### Observed behavior

* use of k9s plugin to reconconcile hr is time consumming as it is currently blocking

### Workaround

The following command displays the command to suspend/resume in batch all failed helm releases
```
flux get helmreleases -A |  awk '{if($5!="True") {print "flux suspend hr " $2 " -n " $1 }}'
flux get helmreleases -A |  awk '{if($5!="True") {print "flux resume hr " $2 " -n " $1 " &" }}'
```

Some (cyclic?) dependencies among helm releases might exist at times:
- nginx helm release is depending on k8s Services to become ready (served by metallb)

### Affected releases

* 56.0.2
* earlier versions

**Comment by gberche-orange** [comment]
Duplicate of https://github.com/orange-cloudfoundry/paas-templates/issues/1581

**Comment by hiddeco** [comment]
Noticed the ping on the annotation PR that introduces `forceAt` and `resetAt`, and want to make a note that the configuration for 2) will change. Instead of enabling it on the controller, you will need to enable it on the `HelmRelease` object, which can be configured with specific ignore rules to e.g. ignore HPA changes.

In addition, instead of cluster drift triggering a Helm upgrade. It will instead create and/or mutate the resources on the cluster directly, taking any ignore rules into account to not reset fields. Which is might lighter, and deals with the issue Helm has of resetting Custom Resources without having to recreate them.

Please refer to the following pull requests for more information. I will be in my dungeon to write proper documentation for the coming days.

- https://github.com/fluxcd/helm-controller/pull/815
- https://github.com/fluxcd/helm-controller/pull/822
